brave/browser-laptop
# Retool the ledger updates into the 0.13.1-branch

Auditor @bsclifton 

Also fixes #6551 of the "sonorious seven"

Test Plan:

1. go to about:preferences#payments
2. set auto-suggest sites to off
3. go to a site in another tab
4. click on the BTC icon
5. go back to the first tab and see if the site is present
